### PR TITLE
fix: web3 account google login with email already used as main email by another account 

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: #425

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: server/__tests__/utils.ts [hunk 1]
```diff
@@ -249,10 +249,8 @@ export async function dummyFilesSetup(
       headers: authHeaders,
       body: form,
     });
-    console.log('resp', resp);
     return { resp };
   } catch (e) {
-    console.log('e', e);
```
Comment (blushi) [inline server/__tests__/utils.ts:255]: just removing some console.log that got merged in unintentionally

**Review by aaronc — COMMENTED** [review]
I'm confused as to why we want to allow this. If someone has account A with that email, why should we let them connect to a different web3 account with the same email but via google instead of email? This seems like something we should disallow.

**Comment by blushi** [comment]
> I'm confused as to why we want to allow this. If someone has account A with that email, why should we let them connect to a different web3 account with the same email but via google instead of email? This seems like something we should disallow.

This is just a consequence of the fact that we let users have a google email that could be different from the main email (used for logging in via OTP) cc/ @erikalogie

**Comment by blushi** [comment]
closing per discussion during the architecture call, we should instead have uniqueness across Google email and main email and when connecting to google, set main email to google email.

**Pull request closed without merging**